core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -1167,6 +1175,141 @@ def load(cls, path: str, reset_paths: bool = True, verbose: bool = True):
                 model.model = model._compiler.load(path=path)
         return model
 
+    def save_learning_curves(self, metrics: str | List[str], curves: dict[dict[str : List[float]]], path: str = None) -> str:
```

Comment:
The model should mark that it saved the learning curves after calling this method. That way it can be part of the model.get_info output as a boolean flag. For example `self.saved_learning_curves = True`.